guides/common/modules/ref_best-practices-for-products-and-repositories.adoc

```diff
@@ -0,0 +1,29 @@
+[id="best-practices-for-products-and-repositories_{context}"]
+= Best practices for products and repositories
+
+* Use one content type per product and content view, for example {client-content-type} content only.
+* Use file repositories for installation media.
+File repositories require a `PULP_MANIFEST` file that you can create using `pulp-manifest /path/to/files`.
+Use local repositories with `\file:///path/to/files` as *Upstream URL* in {Project}, for example for installation media.
+Alternatively, place the installation media file repositories on a web server that can be reached from hosts during provisioning.
+ifdef::orcharhino[]
++
+include::snip_tip-file-repositories-by-atix.adoc[]
+endif::[]
++
+For more information, see xref:Managing_Custom_File_Type_Content_{context}[].
+* Make file repositories available over HTTP.
+If you set *Protected* to true, you can only download content using a global debugging certificate.
+* Use a Hammer script or Ansible Playbook to create a lot of products and repositories when starting with a fresh {Project} installation.
```

Comment:
```suggestion
* Use a Hammer script or Ansible Playbook to create a lot of products and repositories when starting with a fresh {Project} installation.
See the link:https://theforeman.github.io/foreman-ansible-modules/develop/plugins/product_module.html#ansible-collections-theforeman-foreman-product-module[product module] and the link:https://theforeman.github.io/foreman-ansible-modules/develop/plugins/repository_module.html#ansible-collections-theforeman-foreman-repository-module[repository module] in the Foreman Ansible collection.
```

Reply:
If we're going to link to our Ansible modules, an attribute for the base URL would have my preference.

Reply:
Fine with me. Also a lot less work because we can add the same link (or snippet) to each module.

Reply:
Applied.